src/test/scala/Mathematics/AbsMaxSpec.scala

```diff
@@ -0,0 +1,15 @@
+package Mathematics
+
+import org.scalatest.FlatSpec
+
+class AbsMaxSpec extends FlatSpec {
+
+  it should "output the correct Integer as a result from the list of elements" in {
+    assert(AbsMax.absMax(-1000, -1, 999, 72, 65, -56, -767) === -1000)
```

Comment:
the function returns the max absolute value so it should : === 1000.

Reply:
done changes sorry I had previously applied some other logic and this test file is working according to that only. Actually I was returning the original value after finding absolute maximum value but now it is correct.